functions-methods.js

```diff
@@ -33,4 +58,21 @@
 // checkEmailValidity("tessmellink@novi.nl") geeft true - want @ en punt op de juiste plek
 // checkEmailValidity("n.eekenanovi.nl") geeft false - want geen @
 // checkEmailValidity("n.eeken@novinl.") geeft false - want de punt mag niet als laatst
-// checkEmailValidity("tessmellink@novi,nl") geeft false - want er staat een komma in
\ No newline at end of file
+// checkEmailValidity("tessmellink@novi,nl") geeft false - want er staat een komma in
+
+function checkEmailValidity(email) {
+    let valid = false
+    if (email.includes("@")) {
+        if (!(email.includes(",")) && !(email.slice(-1) === ".")) {
+            valid = true
+        }
+    }
```

Comment:
Je kunt deze ook samenvoegen en meteen teruggeven: 
`return email.includes("@") && !email.includes(",") && email.slice(-1) !== ".";`

Reply:
Ok, dat is waar ook, is gecorrigeerd..